.gitignore

```diff
@@ -1,3 +1,4 @@
 *.o
 *.a
 *.so
+/pt-run
```

Comment:
Should this be pt-run wihout the "/" ?

Reply:
It's fine. This is to just ignore `pt-run` binary under the root directory, not all the occurrences of it.

Reply:
Yes, but is there any time when we'd want to not ignore other occurrences of pt-run? Consider that someone might edit the makefile to build pt-run in a different directory instead of the root. Then it might end up accidentally committed to the repo.

Reply:
We also have a directory named pt-run under the src directory. I would like to keep it in a separate directory because currently the src directory is convoluted with Lua and C files. I don't want to change the directory name to something else like `pallene-tracer-run` or move the pt-run source code to src directory anytime soon.

Besides, if someone do happen to create pt-run somewhere else (which they shouldn't), I think they should take responsibility to remove the file before commiting. On the other hand, people might not like the name `pt-run`, edit the Makefile and go with something else entirely. Should we account for that as well? I don't think so. It's a very minor nit, if considered a nit (because Linux binaries tend to not have extensions).

Reply:
That sounds like a convincing argument to get rid of all those sub directories and put everything in src. :)